pkg/mflag/flag.go

```diff
@@ -105,6 +105,44 @@ var ErrRetry = errors.New("flag: retry")
 // -- bool Value
 type boolValue bool
 
+var (
+	m_Daemon_Flags  map[string]bool
+	Has_Daemon_Flag bool
+)
+
+func init() {
+	m_Daemon_Flags = make(map[string]bool)
```

Comment:
actually.... I know our flag setup code is kind of weird, but I wonder if it would be possible to put all daemon flags into a struct/table and then we have code iterate over that table to set all of the flags.  Then we can reference that same table in this code AND in a testcase to validate it all.
Just a thought